packages/itinerary-body/src/AccessLegBody/leg-diagram-preview.tsx

```diff
@@ -121,7 +124,7 @@ class LegDiagramPreview extends Component<Props, State> {
             </S.PreviewDiagramElevationLoss>
           </S.PreviewDiagramTitle>
           {profile.points.length > 0 ? (
-            generateSvg(profile, width)
+            generateSvg(profile, IS_TEST_RUNNER ? "245" : width)
```

Comment:
Note that this is to accommodate tests only, and that no release will be triggered for this package.